pages/whats-new-in-vert-x-4.mdx

```diff
@@ -0,0 +1,59 @@
+import Card from "../components/Card"
+import Cards from "../components/Cards"
+import Layout from "../components/layouts/Intro"
+import Link from "next/link"
+export default Layout
+
+export const meta = {
+  title: "What's new in Vert.x 4"
+}
+
+# What's new in Vert.x 4
+
+Vert.x 4 comes with plenty of new exciting features, some of them have been developed in Vert.x 3 and made available
+as tech previews.
+
+Here is an highlight of the most important features supported in Vert.x 4.
+
+## Future model
+
+Vert.x 4 uses futures over callbacks for modelling asynchronous operations.
```

Comment:
```suggestion
Vert.x 4 uses futures over callbacks for modeling asynchronous operations.
```

Reply:
A bit more details maybe?
I think people may be frightened with this sentence like "Do i need to change all my code using callbacks?"


Suggestion:
```
Vert.x 4 internally uses futures over callbacks for modeling asynchronous operations.
This means you can now use these futures in your code: transform (map) them, compose them, etc. in addition to the other asynchronous programming models Vert.x 3 already used to offer: be it plain callbacks, RxJava bindings, or their expression in the language you picked (continuations, promises, etc.)
```